go.mod

```
@@ -5,4 +5,16 @@ go 1.18
 // Incorrectly published v0.3.0 as a patch (v0.2.6)
 retract v0.2.6
 
+// Incorrectly generated code
+retract v0.3.0
+
+// Incorrectly generated code
+retract v0.3.1
+
+// Incorrectly generated code
+retract v0.3.2
+
+// Incorrectly generated code
+retract v0.3.3
```

Can be simplify as:

```suggestion
// Incorrectly generated code
retract [v0.3.0, v0.3.3]
```

Ref: https://go.dev/ref/mod#go-mod-file-retract